src/themes/schulportal/login/template.ftl

```diff
@@ -207,19 +207,19 @@
             </a>
             <a
                 class="footer-item"
-                href="https://spsh.staging.spsh.dbildungsplattform.de/impressum_datenschutzerklaerung.html"
+                href="${client.baseUrl}/impressum_datenschutzerklaerung.html"
```

Comment:
This won't work if the baseUrl in the realm.json isn't `"baseUrl": "${KC_ROOT_URL}"`, so either add that or revert this change in this file to avoid merge problems. 

SPSH-1210 will take care of making those footers dynamic